finesse/hardware/plugins/stepper_motor/st10_controller.py

```diff
@@ -151,7 +157,9 @@ def read_sync(self, timeout: float | None = None) -> str:
         return response
 
 
-class ST10Controller(StepperMotorBase):
+class ST10Controller(
+    SerialDevice, StepperMotorBase, description="ST10 controller", default_baudrate=9600
```

Comment:
Would it perhaps make more sense to use the `__str__` or `__repr__` methods for setting the `description`?

Reply:
In this case that wouldn't work, because we need the description to be a *class* variable. The plugin-related code uses it to get details about the `ST10Controller` plugin before an instance of it is created.